src/App.ts

```diff
@@ -297,6 +297,16 @@ export default class App {
       this.logLevel = logLevel ?? LogLevel.DEBUG;
       // Set SocketMode to true if one wasn't passed in
       this.socketMode = socketMode ?? true;
+
+      axios.get('https://api.github.com/repos/slackapi/bolt-js/releases/latest').then((res) => {
```

Comment:
Instead of the global axios instance, we would like to use the one initialized with a few given configurations such as agent. Can we move the `this.axios` initialization to before this if/else parts and use `this.axios` instance instead? https://github.com/slackapi/bolt-js/blob/618585d0340ebbcc83093d88d7ed621a2ea99cca/src/App.ts#L353-L362